src/bundle/Resources/views/content/tab/versions/table.html.twig

```diff
@@ -36,7 +36,7 @@
                 'contentId': version.contentInfo.id,
                 'versionNo': version.versionNo,
                 'language': version.initialLanguageCode,
-                'locationId': location.id
+                'locationId': location.id ?? null
```

Comment:
This will not throw an exception when location is null?

Reply:
I tested it 
```
{% set location = null %}
{{ dump(location.id ?? null) }}
```
end result is `null`